core/domain/feedback_services_test.py

```diff
@@ -232,7 +232,52 @@ def test_get_all_threads_without_suggestion(self):
             with self.swap(utils, 'get_time_in_millisecs',
                            self._get_milliseconds):
                 threads = feedback_services.get_all_threads(
-                    self.EXP_ID2, False)
+                    self.EXP_ID1, False)
```

Comment:
I am not against using self.swap, but for curiosity: 
Why we used self.swap to get_threads?
Will it be better to create threads actually and test it?
I think, this tests were wrong logically.